src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -25,6 +25,7 @@ class InputOption
     const VALUE_REQUIRED = 2;
     const VALUE_OPTIONAL = 4;
     const VALUE_IS_ARRAY = 8;
+    const VALUE_IS_COMMA_SEPARATED = 16;
```

Comment:
Shouldn't the value directly be `24` in order to have the `no VALUE_IS_COMMA_SEPARATED without VALUE_IS_ARRAY` condition always satisfied ?